core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -99,34 +121,113 @@ angular.module('oppia').controller('TranslationModalController', [
       $scope.activeWrittenTranslation.html = '';
     };
 
+    $scope.hasTranslationErrors = function(data, behaviour) {
+      return data.some(state => state === behaviour);
+    };
+
+    $scope.findImgAttributes = function(elements, type) {
+      var attributes = [];
```

Comment:
If a file is already using `const` and `let`, then try not using `var` as it can lead to confusing bugs. Also, can't we combine the declaration of `attributes` and its assignment on line 131?

Reply:
Done